exercise_html/home.html

```diff
@@ -0,0 +1,418 @@
+                    alt="pet"
+                    class="nav-item_icon"
+                  /><span>Pets</span></a
+                >
+              </li>
+              <li>
+                <a href="#"
+                  ><img
+                    src="./assets/images/sknife-folk.svg"
+                    alt="kitchen"
+                    class="nav-item_icon"
+                  /><span>Kitchen Accessories</span></a
+                >
+              </li>
+            </ul>
+          </nav>
+        </div>
+      </div>
+    </header>
+    <main>
+      <section class="banner">
+        <img
+          src="./assets/images/banner.jpg"
+          alt="this is the banner"
+          class="banner-image"
+        />
+      </section>
+      <section class="product">
+        <section class="product-control space-between">
+          <div class="product-control-left">
+            <div>
+              <span>SORT BY</span
+              ><select name="sort-by" id="sort-by">
+                <option value="useless" selected>Useless</option>
+              </select>
+            </div>
+            <div>
+              <select name="condition" id="condition">
+                <option value="useless" selected>Condition</option>
+              </select>
+            </div>
+            <div>
+              <select name="delivery-option" id="delivery-option">
+                <option value="useless" selected>Delivery Option</option>
+              </select>
+            </div>
+          </div>
+          <div class="product-control-right">
+            <div>
+              <ul>
+                <li><button>Show all</button></li>
+                <li><button>Auction</button></li>
+                <li><button>Buy now</button></li>
+              </ul>
+            </div>
+            <div>
+              <ul class="product-view-option">
+                <li class="product-view-option__list">
+                  <span><img src="./assets/images/list.svg" alt="list" /></span>
+                </li>
+                <li class="product-view-option__grid">
+                  <span><img src="./assets/images/grid.svg" alt="grid" /></span>
+                </li>
+              </ul>
+            </div>
+          </div>
+        </section>
+        <section class="products-container">
+          <ul class="product-list">
+            <li class="product-item">
+              <a href="#"
+                ><figure>
+                  <img
+                    src="./assets/images/product-1.png"
+                    alt="this is the product image"
+                    class="product-item__image"
+                  />
+                  <figcapture class="product-item_image-description"
+                    >Vintage Typewriter to post awesome stories about UI design
+                    and webdev.</figcapture
+                  >
+                </figure></a
+              >
+
+              <h2 class="product-item__price">$49.50</h2>
```

Comment:
ko xài thẻ `heading` tại đây. dùng heading phải đúng chức năng. ko hẳn là cứ chữ to thì mình sẽ xài heading đâu. ở đâu xài thẻ `<span>` bth thôi